src/components/iOSThirdPartyData.svelte

```diff
@@ -0,0 +1,147 @@
+<script context="module">
+  export const adjustDataTypes = {
+    // used for label description
+    Advertising: "Information about Advertising",
+    Installation: "Information about how the application was installed",
+    Application:
+      "Information about the version and build of the application and its bundle ID",
+    Device: "Information about the make and model of your device",
+    User: "Information about the user",
+  };
+</script>
+
+<script>
+  import ItemList from "./ItemList.svelte";
+
+  // the Adjust data is hard-coded here as only Firefox iOS requires it atm
+  // when the need grows then we can start abstracting it
+  const adjustData = [
+    {
+      name: "adid",
+      description:
+        "The advertising identifier, specific to Mozilla, and managed using Apple's skAdNetwork API",
+      types: ["Advertising"],
+    },
+    {
+      name: "attribution_deeplink",
+      description: "Used to attribute a deeplink campaign",
+      types: ["Advertising"],
+    },
+    {
+      name: "engagement_type",
+      description:
+        "Used to attribute to a campaign that would otherwise be considered organic",
+      types: ["Advertising"],
+    },
+    {
+      name: "reference_tag",
+      description: "The click ID for the campaign",
+      types: ["Advertising"],
+    },
+    {
+      name: "tracker_token",
+      description: "Linking the install to the ads exposed",
+      types: ["Advertising"],
+    },
+    {
+      name: "event_token",
+      description: "Information about how the application was installed",
+      types: ["Installation"],
+    },
+    {
+      name: "event_cost_id",
+      description: "Information about how the application was installed",
+      types: ["Installation"],
+    },
+    {
+      name: "store_name",
+      description: "Information about how the application was installed",
+      types: ["Installation"],
+    },
+    {
+      name: "app_name",
+      description: "",
+      types: ["Application"],
+    },
+    {
+      name: "app_token",
+      description: "",
```

Comment:
the description is empty for certain metrics because no description was provided for them in the original Google docs. I'm wondering if leaving the description blank is okay for now @mdboom?

Reply:
Yep -- leaving them empty for now is fine.